internal/js/modules/k6/browser/common/element_handle.go

```diff
@@ -1673,24 +1690,38 @@ func (h *ElementHandle) newPointerAction(
 func retryPointerAction(
 	apiCtx context.Context, fn retryablePointerActionFunc, opts *ElementHandleBasePointerOptions,
 ) (res any, err error) {
-	// try the default scrolling
-	if res, err = fn(apiCtx, nil); opts.Force || err == nil {
-		return res, err
-	}
-	// try with different scrolling options
-	for _, p := range []ScrollPosition{
-		ScrollPositionStart,
-		ScrollPositionCenter,
-		ScrollPositionEnd,
-		ScrollPositionNearest,
-	} {
-		s := ScrollIntoViewOptions{Block: p, Inline: p}
-		if res, err = fn(apiCtx, &s); err == nil {
-			break
+	for {
+		res, err = fn(apiCtx, nil)
+		if opts.Force || err == nil {
+			return res, err
 		}
-	}
 
-	return res, err
+		// try with different scrolling options
+		for _, p := range []ScrollPosition{
+			ScrollPositionStart,
+			ScrollPositionCenter,
+			ScrollPositionEnd,
+			ScrollPositionNearest,
+		} {
+			s := ScrollIntoViewOptions{Block: p, Inline: p}
+			if res, err = fn(apiCtx, &s); err == nil {
+				return res, nil
+			}
+		}
+
+		if !strings.Contains(err.Error(), "check if element is visible") &&
```

Comment:
Same as above

Reply:
Done in e5ce6ae7cdf2affc08cffe68acf45d5cb14d4278